cmake/BemppFindDependencies.cmake

```diff
@@ -28,7 +28,7 @@ find_package(Doxygen)
 find_package(CoherentPython REQUIRED)
 # Get Python home directory
 include(CallPython)
-call_python(PYTHON_HOME "import sys; print(sys.prefix)")
+call_python(PYTHON_HOME "import sys; print('%s:%s' % (sys.base_prefix, sys.base_exec_prefix))")
```

Comment:
Does this work in python 2? https://docs.python.org/3/library/sys.html#sys.exec_prefix